--- a/include/asm-generic/pgtable.h     2023-01-24 02:09:49.000000000 +0000
+++ b/include/asm-generic/pgtable.h     2023-02-15 18:49:44.237072881 +0000
@@ -924,10 +924,12 @@ static inline bool pmd_special(pmd_t pmd
        return false;
 }
 
+/*
 static inline pmd_t pmd_mkspecial(pmd_t pmd)
 {
        return pmd;
 }
+*/
 #endif
 
 #if !defined(CONFIG_TRANSPARENT_HUGEPAGE) || \
@@ -941,10 +943,12 @@ static inline bool pud_special(pud_t pud
 {
        return false;
 }
+/*
 static inline pud_t pud_mkspecial(pud_t pud)
 {
        return pud;
 }
+*/
 #endif
 
 /* See pmd_none_or_trans_huge_or_clear_bad for discussion. */
